tests/unittests/execution_store_tests.py

```
@@ -0,0 +1,226 @@
+"""
+FiftyOne execution store related unit tests.
+
+| Copyright 2017-2024, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+import time
+import unittest
+from unittest.mock import patch, MagicMock, ANY, Mock
```

Ruff

`unittest.mock.patch` imported but unused

Remove unused import

(F401)